source/jormungandr/jormungandr/instance.py

```diff
@@ -47,7 +47,7 @@
 from flask import g
 import flask
 import pybreaker
-from jormungandr import georef, planner, schedule
+from jormungandr import georef, planner, schedule, rt_schedule
```

Comment:
I think we should write rt in full, "realtime" would be easier to read in a few months.